app/controllers/books_controller.rb

```diff
@@ -12,9 +12,13 @@ def edit
   end
 
   def create
-    @book = Book.create(book_params)
-    flash[:success] = t('books.create.flash_success')
-    redirect_to books_path
+    @book = Book.new(book_params)
```

Comment:
@paulcsmith, I refactored the `create` function to keep the user on the page until a valid book (one containing both an author and a unique title is entered). Previously, the book would not be saved and the user would not be notified, but would instead be redirected to `books_path` immediately with no notification of what went wrong.